src/pages/home/report/ReportActionItem.js

```diff
@@ -138,6 +141,12 @@ function ReportActionItem(props) {
     const prevDraftMessage = usePrevious(props.draftMessage);
     const originalReportID = ReportUtils.getOriginalReportID(props.report.reportID, props.action);
     const originalReport = props.report.reportID === originalReportID ? props.report : ReportUtils.getReport(originalReportID);
+    const reportActionID = lodashGet(route, 'params.reportActionID');
+
+    const highlightedBackgroundColorIfNeeded = useMemo(
+        () => (reportActionID === props.action.reportActionID ? {backgroundColor: themeColors.highlightBG} : {}),
```

Comment:
We don't prefer inline style.
```suggestion
        () => (reportActionID === props.action.reportActionID ? StyleUtils.getBackgroundColorStyle(themeColors.highlightBG) : {}),
```

Reply:
👍🏻